getting-help/service-section.html

```diff
@@ -1,3 +1,12 @@
+{% if jekyll.environment == "production" %}
+<script>
+    window.mode = "production";
+</script>
+{% else %}
+<script>
+    window.mode = "development";
+</script>
+{% endif %}
```

Comment:
So we have the frontmatter Jekyll tags in the JS code, do we still need this config?